core/src/modules/ops/group.ts

```diff
@@ -96,18 +104,25 @@ export namespace GroupOps {
         }
       }
 
-      if (bulkCreateRecordIds.length > 0) {
+      while (bulkCreateRecordIds.length > 0) {
         await GroupMember.bulkCreate(
-          bulkCreateRecordIds.map((recordId) => {
-            return { recordId, groupId: group.id };
-          })
+          bulkCreateRecordIds
+            .splice(0, config.batchSize.internalWrite)
+            .map((recordId) => {
+              return { recordId, groupId: group.id };
+            })
         );
       }
```

Comment:
In a few places we are now trying to respect `config.batchSize.internalWrite` more by only updating 1000 records/rows at a time